completion generation is broken

Completion generation for 0.88.0 release build is broken as error below:

```
# ex: ts=4 sw=4 et filetype=sh
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1046da8f4]

goroutine 13 [running]:
github.com/chainloop-dev/chainloop/app/cli/internal/telemetry/posthog.(*Tracker).TrackEvent(0x0, {0x14000e00030?, 0x1046fbb76?}, {0x10470d2dc, 0x10}, {0x14000db00c0, 0x40}, 0x14000e00030)
	github.com/chainloop-dev/chainloop/app/cli/internal/telemetry/posthog/posthog.go:56 +0x44
github.com/chainloop-dev/chainloop/app/cli/internal/telemetry.(*CommandTracker).Track(0x14000e1bf48, {0x1055f2ff8, 0x106af4360}, {0x1400082a020, 0xf}, 0x14000d91f18)
	github.com/chainloop-dev/chainloop/app/cli/internal/telemetry/telemetry.go:68 +0x31c
github.com/chainloop-dev/chainloop/app/cli/cmd.recordCommand(0x14000d2c008, 0x0)
	github.com/chainloop-dev/chainloop/app/cli/cmd/root.go:367 +0x364
github.com/chainloop-dev/chainloop/app/cli/cmd.NewRootCmd.func1.1()
	github.com/chainloop-dev/chainloop/app/cli/cmd/root.go:118 +0xc8
created by github.com/chainloop-dev/chainloop/app/cli/cmd.NewRootCmd.func1 in goroutine 1
	github.com/chainloop-dev/chainloop/app/cli/cmd/root.go:107 +0x340
```

relates to https://github.com/Homebrew/homebrew-core/pull/172169

--- a/app/cli/internal/telemetry/posthog/posthog.go
+++ b/app/cli/internal/telemetry/posthog/posthog.go
@@ -53,6 +53,10 @@ type Tracker struct {
 
 // TrackEvent sends an event to the PostHog server.
 func (p *Tracker) TrackEvent(_ context.Context, eventName string, id string, tags telemetry.Tags) error {
+	if p == nil {
+		return nil
+	}
+
 	defer p.client.Close()
 	msg := posthog.Capture{
 		DistinctId: id,

